src/manager.rs

```diff
@@ -66,7 +66,8 @@ impl<Env, T> ScriptManager<Env, T>
         try!(connection.execute("CREATE TABLE IF NOT EXISTS scripts (
             id          TEXT NOT NULL PRIMARY KEY,
             source      TEXT NOT NULL,
-            is_enabled  BOOL NOT NULL DEFAULT 1
+            is_enabled  BOOL NOT NULL DEFAULT 1,
+            owner       INTEGER NOT NULL DEFAULT -1
```

Comment:
Can you add documentation? All other fields of the database are basically self-documenting, but that one really isn't.